docs/release/release-process.md

```diff
@@ -20,14 +20,14 @@ Actions](https://github.com/features/actions) for pipelines.
    branch (typically documentation).
 1. **tag release**: Run the `Tag` action on the _release branch_ with the
    desired version (e.g. `v0.14.0`).
-1. **tag next pre-release**: Run the `tag` action on the `master` branch with
-   the `rc.0` for the next release (e.g. `v0.15.0-rc.0`).
 1. **build/publish**: Run the `CI` action on the release branch with the version
    that was just tagged.
 1. **verify**: Verify all artifacts have been published successfully, perform
    sanity testing.
 1. **promote**: Run the `Promote` action to promote release to desired
    channel(s).
+1. **tag next pre-release**: Run the `tag` action on the `master` branch with
```

Comment:
@muvaf i think we can put this after the build/publish step right? In the promote job we specify the already built tag we want to promote, so it's okay if this one exists already at that point

Reply:
done!